src/routes/(site)/wave2/+page.svelte

```diff
@@ -0,0 +1,176 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { MetaTags } from 'svelte-meta-tags';
+  import Block from '$lib/components/block/block.svelte';
+  import Button from '$lib/components/button/button.svelte';
+  import Spacing from '$lib/public/legacy/spacing.svelte';
+  import { TextAlignment } from '$lib/public/text-alignment/text-alignment';
+  import PositionList from './position-list.svelte';
+  import { POSITIONS } from './data';
+
+  /**
+   * positions is the list of position elements.
+   */
+  let positions: HTMLDialogElement[] | null = null;
+
+  /**
+   * expanded is true when every position is open.
+   */
+  let expanded = false;
+
+  /**
+   * checkExpanded tests whether a set of positions are expanded.
+   */
+  function checkExpanded() {
+    expanded = positions !== null && positions.every((el) => el.hasAttribute('open'));
+  }
+
+  /**
+   * action is done each time the user clicks the primary button.
+   */
+  function action() {
+    if (positions === null) {
+      return;
+    }
+
+    // Defers checkExpanded for after this function is called,
+    // no matter the path this function continues to take.
+    setTimeout(checkExpanded);
+
+    if (expanded) {
+      positions.forEach((el) => el.removeAttribute('open'));
+      return;
+    }
+
+    positions.forEach((el) => el.setAttribute('open', 'true'));
+  }
+
+  onMount(() => {
+    positions = [...document.querySelectorAll<HTMLDialogElement>('.position')];
+  });
+</script>
+
+<svelte:head>
+  <title>Wave 1 board applications | ACM at CSUF</title>
+</svelte:head>
+
+<MetaTags
+  openGraph={{
+    title: 'Fall 2023 board applications',
+    description:
+      'Listed below are the positions that are open for the Fall 2023 semester. Please read the descriptions carefully and apply for the position(s) that you are interested in. You may apply for multiple positions, but you may only be selected for one. If you are selected for a position, you will be contacted by the current board member in charge of that position.',
+    url: 'https://acmcsuf.com/wave2',
+    type: 'article',
+    article: {
+      publishedTime: '2023-22-05T00:00:00.000Z',
+      modifiedTime: '2023-22-05T00:00:00.000Z',
+    },
+  }}
+/>
+
+<Spacing --min="175px" --med="200px" --max="200px" />
+
+<Block align={TextAlignment.LEFT}>
+  <h1 slot="headline" class="size-lg">Wave 1 board applications</h1>
```

Comment:
```suggestion
  <h1 slot="headline" class="size-lg">Wave 2 board applications</h1>
```

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/861/commits/b69e665346ec42f2bc26071acfa52a77fc74d06d>.